rust/operator-binary/src/config.rs

```diff
@@ -561,6 +571,47 @@ pub fn build_nifi_properties(
         "nifi.cluster.protocol.is.secure".to_string(),
         "true".to_string(),
     );
+
+    if let NifiAuthenticationConfig::Oidc { provider, oidc, .. } = auth_config {
+        let endpoint_url = provider
+            .endpoint_url()
+            .context(InvalidOidcEndpointSnafu)?
+            .to_string();
+        properties.insert(
+            "nifi.security.user.oidc.discovery.url".to_string(),
+            format!("{endpoint_url}/{DEFAULT_OIDC_WELLKNOWN_PATH}"),
```

Comment:
Nit: The [docs](https://github.com/stackabletech/operator-rs/blob/a796eed0fb4a70ced54027662fc7174b8474bce2/crates/stackable-operator/src/commons/authentication/oidc.rs#L112-L114) suggest to use `url.join(DEFAULT_OIDC_WELLKNOWN_PATH)`. But watch out, this behaves a bit strange sometimes (in relation to slashes)